service/src/main/resources/application.yml

```diff
@@ -86,15 +86,11 @@ otel:
   sdk:
     disabled: false # set to true to disable all open telemetry features
 
-  springboot:
-    resource:
-      attributes:
-        service:
-          name: ${spring.application.name}
-
-  instrumentation:
-    logback-appender:
-      enabled: false
+  resource:
```

Comment:
This change is copied from https://github.com/DataBiosphere/terra-billing-profile-manager/pull/535